For these checks, use the libraries in the proto area, in case we add a new
symbol to libc or friends that gnupg's `configure` picks up.

diff -ru /tmp/gnupg-1.4.11/checks/defs.inc /tmp/gnupg-1.4.11-OS-8313-2//checks/defs.inc
--- /tmp/gnupg-1.4.11/checks/defs.inc   Thu Aug 13 03:29:18 2009
+++ /tmp/gnupg-1.4.11-OS-8313-2//checks/defs.inc        Wed Aug 25 01:06:48 2021
@@ -31,6 +31,10 @@
 LC_ALL=
 LC_MESSAGES=
 
+# As part of a smartos-live build, set the LD_LIBRARY_PATH to the smartos-live
+# proto area.
+LD_LIBRARY_PATH=$PWD/../../../../../proto/usr/lib
+
 # Internal use.
 defs_stop_on_error=no
 defs_error_seen=no
